mars/services/task/execution/ray/context.py

```diff
@@ -155,6 +155,9 @@ def get_worker_addresses(self) -> List[str]:
         # Returns virtual worker addresses.
         return self._worker_addresses
 
+    def storage_put(self, key: str = None, obj=None):  # pragma: no cover
```

Comment:
Maybe we could add some tests to cover this method and `storage_get` is the same.